src/lib/components/TeamKeys.svelte

```diff
@@ -36,7 +39,13 @@
       disabled={userTeam.role === 'MEMBER'}
       value={form?.fields?.openAiApiKey ?? team.openAiApiKey}
       errors={form?.errors?.openAiApiKey}
+      on:input={(event) => {
+        testApiKey = event.detail.target?.value
+      }}
     />
+    <div class="col-span-full">
+      <TestApiButton {testApiKey} />
+    </div>
```

Comment:
Some svelte magic:

```suggestion
  <TestApiButton class="col-span-full" {testApiKey} />
```

and then you do inside the component `<div class="flex xyz {$$props.class}">`

Reply:
Fancy!! I'll make the changes asap ✨